allensdk/brain_observatory/behavior/behavior_ophys_session.py

```diff
@@ -67,22 +67,4 @@ def __eq__(self, other):
 
 if __name__ == "__main__":
 
-    session = BehaviorOphysSession(789359614)
```

Comment:
I would like to keep this in here for debugging.

Reply:
np